### PR TITLE
Update: Korean language pack for CI v4.1.2

### PR DESCRIPTION
**Pull request by Timothy-Y-H-Lee**
1. Fix typo in View.php
2. Update: Korean language pack for CI v4.1.2

### CODE REVIEW

**Review by paulbalandan — CHANGES_REQUESTED** [review]
Please fix the CS violations and the unresolved merge conflicts.

**Comment by paulbalandan** [comment]
You can run `vendor/bin/php-cs-fixer fix --verbose` to automatically fix the CS.

**Comment by Timothy-Y-H-Lee** [comment]
Fixed codes by php-cs-fixer.
Thank you for your information.

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ claims no expertise beyond English.
 | id     | Indonesian           | 4.0.0   | [ridho1991][id]      | Mutasim Ridlo
 | it     | Italian              | 4.0.0   | [luk3b8][it]         | Luca Rasia
 | ja     | Japanese             | 4.0.3   | [kenjis][ja]         | Kenji Suzuki
-| ko     | Korean               | 4.0.2   | [codeigniter-kr][kr] | codeigniter-kr
+| ko     | Korean               | 4.1.2   | [codeigniter-kr][kr] | codeigniter-kr
```
Comment (paulbalandan) [inline README.md:45]: Please revert this one. Version releases pertain to the version number where a locale is **first** introduced. So this should not change.

Comment (Timothy-Y-H-Lee) [reply README.md]: I have just fixed it. :)

**Review by paulbalandan — APPROVED** [review]

**Comment by paulbalandan** [comment]
@Timothy-Y-H-Lee thank you

**Pull request merged**